frontend/packages/ceph-storage-plugin/src/components/namespace-store/namespace-store-dropdown.tsx

```diff
@@ -117,7 +111,7 @@ export const NamespaceStoreDropdown: React.FC<NamespaceStoreDropdownProps> = ({
             data-test="nns-dropdown-toggle"
             onToggle={() => setOpen(!isOpen)}
           >
-            {selectedKey || nsName || t('ceph-storage-plugin~Select a namespace store')}
+            {selectedKey || t('ceph-storage-plugin~Select a namespace store')}
```

Comment:
```suggestion
            {selectedKey || t('ceph-storage-plugin~Select a NamespaceStore')}
```
isn't it?

Reply:
we are using the same style in backing store dropdown. Lets keep it same. I am planning to make the component generic.